src/widget/rating/rating.js

```diff
@@ -0,0 +1,117 @@
+import RangeWidget from '../../widget/range/range-widget';
+import { isNumber } from '../../js/utils';
+
+/**
+ * @extends RangeWidget
+ */
+class RatingWidget extends RangeWidget {
+    /**
+     * @type string
+     */
+    static get selector() {
+        return '.or-appearance-rating input[type="number"]';
+    }
+
+    _init() {
+        const fragment = document.createRange().createContextualFragment( this._getHtmlStr() );
+        this.element.after( fragment );
+        this.element.classList.add( 'hide' );
+        this.widget = this.question.querySelector( '.widget' );
+
+        let ticks = this.props.ticks ? Math.ceil( Math.abs( ( this.props.max - this.props.min ) / this.props.step ) ) : 1;
```

Comment:
this.props.ticks is not relevant for the rating widget, so that check can be removed. Stars will always be shown.

Reply:
code removed